### ISSUE TITLE
[2.1.4] iPhone 6S / 10.1.1 - Multiple Like for lomotif without login - Search for any user -> open lomotif and click on Like multiple time

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#1233]
Frequency: Every time

Pre-requisite: User should be logged out from lomotif app

Action:
1. Launch the app
2. Tap on user search icon
3. Search for any user and open his/her profile
4. Tap on any existing lomotif to play it
5. During playing, click on Like (heart) symbol
6. Like count will be increased by 1 and also there is a popup display "Sign up / log in to follow your friends."
7. Click on "OK" on popup message
8. Again click on Like symbol
9. Check whether Count increase multiple times or not

Expected result: Without login, if Like event works then it should allow user to like any lomotif only once

Actual result: User can like lomotif multiple times without login in to lomotif app

**Comment by Pankaj-Testershub** [comment]
@xiuie Duplicate of #1196.
